Wikipedia/Code/UILabel+WMFAuth.swift

```diff
@@ -0,0 +1,13 @@
+
+extension UILabel {
+    /// Makes an attributed string from two strings performing a **$1** substitution and applying a common color scheme (black/blue) used by a few auth view controllers.
+    /// Also re-uses the label's font so the resulting attributed string respects storyboard dynamic type font choice and responds to dynamic type size changes.
+    ///
+    /// - Parameters:
+    ///   - dollarSignString: A localization string which has a **$1** subtitution for the string from the **substitutionString** parameter.
+    ///   - substitutionString: The string which will replace **$1** in the **dollarSignString** parameter.
+    /// - Returns: Attributed string from the two parameter strings, with the **dollarSignString** in black and the **substitutionString** in blue.
+    func wmf_authAttributedStringReusingFont(withDollarSignString dollarSignString: String, substitutionString: String) -> NSAttributedString {
```

Comment:
@amrox 

Sure! It's used in 3 places:

![untitled mov](https://cloud.githubusercontent.com/assets/3143487/23565603/208dd842-0003-11e7-87dc-7994ddf4bf25.gif)

For the login place it says "Don't have an account? Join Wikipedia.", which comes from:
> "login-no-account" = "Don't have an account? $1";
> "login-join-wikipedia" = "Join Wikipedia.";

I did these as 2 separate strings so those parts could be styled independently.

One other thing to keep in mind is we generally avoid simply appending strings which are to be combined in some way, as in this case. "$1" is used instead so translators can choose a language appropriate combination.